[2021-12-19 12:34:38,536] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: scraping_daft_longford.scrap_data scheduled__2021-12-18T10:10:00+00:00 [queued]>
[2021-12-19 12:34:38,833] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: scraping_daft_longford.scrap_data scheduled__2021-12-18T10:10:00+00:00 [queued]>
[2021-12-19 12:34:38,836] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-19 12:34:38,839] {taskinstance.py:1242} INFO - Starting attempt 1 of 4
[2021-12-19 12:34:38,841] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-19 12:34:38,960] {taskinstance.py:1262} INFO - Executing <Task(SparkSubmitOperator): scrap_data> on 2021-12-18 10:10:00+00:00
[2021-12-19 12:34:39,086] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'scraping_daft_longford', 'scrap_data', 'scheduled__2021-12-18T10:10:00+00:00', '--job-id', '3190', '--raw', '--subdir', 'DAGS_FOLDER/scraping_daft_longford.py', '--cfg-path', '/tmp/tmp4hh62ltz', '--error-file', '/tmp/tmpzow956gw']
[2021-12-19 12:34:39,097] {standard_task_runner.py:77} INFO - Job 3190: Subtask scrap_data
[2021-12-19 12:34:39,033] {standard_task_runner.py:52} INFO - Started process 2302 to run task
[2021-12-19 12:34:40,264] {logging_mixin.py:109} INFO - Running <TaskInstance: scraping_daft_longford.scrap_data scheduled__2021-12-18T10:10:00+00:00 [running]> on host 358a7891c658
[2021-12-19 12:34:42,123] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=clement.liscoet@gmail.com
AIRFLOW_CTX_DAG_OWNER=clemoni
AIRFLOW_CTX_DAG_ID=scraping_daft_longford
AIRFLOW_CTX_TASK_ID=scrap_data
AIRFLOW_CTX_EXECUTION_DATE=2021-12-18T10:10:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-18T10:10:00+00:00
[2021-12-19 12:34:42,201] {base.py:79} INFO - Using connection to: id: spark_default. Host: spark://spark:7077, Port: None, Schema: , Login: ***, Password: ***, extra: {}
[2021-12-19 12:34:42,209] {spark_submit.py:360} INFO - Spark-Submit cmd: spark-submit --master spark://spark:7077 --conf spark.master=spark://spark:7077 --name scrap data --verbose /opt/***/jobs/get_ads_per_county.py longford
[2021-12-19 12:35:20,177] {spark_submit.py:514} INFO - Using properties file: null
[2021-12-19 12:35:22,758] {spark_submit.py:514} INFO - WARNING: An illegal reflective access operation has occurred
[2021-12-19 12:35:22,759] {spark_submit.py:514} INFO - WARNING: Illegal reflective access by org.apache.spark.unsafe.Platform (file:/home/***/.local/lib/python3.6/site-packages/pyspark/jars/spark-unsafe_2.12-3.2.0.jar) to constructor java.nio.DirectByteBuffer(long,int)
[2021-12-19 12:35:22,760] {spark_submit.py:514} INFO - WARNING: Please consider reporting this to the maintainers of org.apache.spark.unsafe.Platform
[2021-12-19 12:35:22,773] {spark_submit.py:514} INFO - WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
[2021-12-19 12:35:22,774] {spark_submit.py:514} INFO - WARNING: All illegal access operations will be denied in a future release
[2021-12-19 12:35:23,249] {spark_submit.py:514} INFO - Parsed arguments:
[2021-12-19 12:35:23,256] {spark_submit.py:514} INFO - master                  spark://spark:7077
[2021-12-19 12:35:23,294] {spark_submit.py:514} INFO - deployMode              null
[2021-12-19 12:35:23,306] {spark_submit.py:514} INFO - executorMemory          null
[2021-12-19 12:35:23,308] {spark_submit.py:514} INFO - executorCores           null
[2021-12-19 12:35:23,314] {spark_submit.py:514} INFO - totalExecutorCores      null
[2021-12-19 12:35:23,319] {spark_submit.py:514} INFO - propertiesFile          null
[2021-12-19 12:35:23,320] {spark_submit.py:514} INFO - driverMemory            null
[2021-12-19 12:35:23,325] {spark_submit.py:514} INFO - driverCores             null
[2021-12-19 12:35:23,327] {spark_submit.py:514} INFO - driverExtraClassPath    null
[2021-12-19 12:35:23,338] {spark_submit.py:514} INFO - driverExtraLibraryPath  null
[2021-12-19 12:35:23,347] {spark_submit.py:514} INFO - driverExtraJavaOptions  null
[2021-12-19 12:35:23,355] {spark_submit.py:514} INFO - supervise               false
[2021-12-19 12:35:23,361] {spark_submit.py:514} INFO - queue                   null
[2021-12-19 12:35:23,371] {spark_submit.py:514} INFO - numExecutors            null
[2021-12-19 12:35:23,388] {spark_submit.py:514} INFO - files                   null
[2021-12-19 12:35:23,407] {spark_submit.py:514} INFO - pyFiles                 null
[2021-12-19 12:35:23,410] {spark_submit.py:514} INFO - archives                null
[2021-12-19 12:35:23,414] {spark_submit.py:514} INFO - mainClass               null
[2021-12-19 12:35:23,425] {spark_submit.py:514} INFO - primaryResource         file:/opt/***/jobs/get_ads_per_county.py
[2021-12-19 12:35:23,432] {spark_submit.py:514} INFO - name                    scrap data
[2021-12-19 12:35:23,438] {spark_submit.py:514} INFO - childArgs               [longford]
[2021-12-19 12:35:23,440] {spark_submit.py:514} INFO - jars                    null
[2021-12-19 12:35:23,444] {spark_submit.py:514} INFO - packages                null
[2021-12-19 12:35:23,447] {spark_submit.py:514} INFO - packagesExclusions      null
[2021-12-19 12:35:23,450] {spark_submit.py:514} INFO - repositories            null
[2021-12-19 12:35:23,455] {spark_submit.py:514} INFO - verbose                 true
[2021-12-19 12:35:23,459] {spark_submit.py:514} INFO - 
[2021-12-19 12:35:23,461] {spark_submit.py:514} INFO - Spark properties used, including those specified through
[2021-12-19 12:35:23,465] {spark_submit.py:514} INFO - --conf and those from the properties file null:
[2021-12-19 12:35:23,469] {spark_submit.py:514} INFO - (spark.master,spark://spark:7077)
[2021-12-19 12:35:23,473] {spark_submit.py:514} INFO - 
[2021-12-19 12:35:23,475] {spark_submit.py:514} INFO - 
[2021-12-19 12:35:27,320] {spark_submit.py:514} INFO - Main class:
[2021-12-19 12:35:27,322] {spark_submit.py:514} INFO - org.apache.spark.deploy.PythonRunner
[2021-12-19 12:35:27,323] {spark_submit.py:514} INFO - Arguments:
[2021-12-19 12:35:27,325] {spark_submit.py:514} INFO - file:/opt/***/jobs/get_ads_per_county.py
[2021-12-19 12:35:27,326] {spark_submit.py:514} INFO - null
[2021-12-19 12:35:27,327] {spark_submit.py:514} INFO - longford
[2021-12-19 12:35:27,328] {spark_submit.py:514} INFO - --verbose
[2021-12-19 12:35:27,341] {spark_submit.py:514} INFO - Spark config:
[2021-12-19 12:35:27,343] {spark_submit.py:514} INFO - (spark.app.name,scrap data)
[2021-12-19 12:35:27,344] {spark_submit.py:514} INFO - (spark.master,spark://spark:7077)
[2021-12-19 12:35:27,345] {spark_submit.py:514} INFO - (spark.submit.pyFiles,)
[2021-12-19 12:35:27,347] {spark_submit.py:514} INFO - (spark.submit.deployMode,client)
[2021-12-19 12:35:27,348] {spark_submit.py:514} INFO - Classpath elements:
[2021-12-19 12:35:27,349] {spark_submit.py:514} INFO - 
[2021-12-19 12:35:27,351] {spark_submit.py:514} INFO - 
[2021-12-19 12:35:27,352] {spark_submit.py:514} INFO - 
[2021-12-19 12:35:34,458] {spark_submit.py:514} INFO - Using Spark's default log4j profile: org/apache/spark/log4j-defaults.properties
[2021-12-19 12:35:34,574] {spark_submit.py:514} INFO - 21/12/19 12:35:34 INFO SparkContext: Running Spark version 3.2.0
[2021-12-19 12:35:35,464] {spark_submit.py:514} INFO - 21/12/19 12:35:35 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2021-12-19 12:35:36,788] {spark_submit.py:514} INFO - 21/12/19 12:35:36 INFO ResourceUtils: ==============================================================
[2021-12-19 12:35:36,791] {spark_submit.py:514} INFO - 21/12/19 12:35:36 INFO ResourceUtils: No custom resources configured for spark.driver.
[2021-12-19 12:35:36,793] {spark_submit.py:514} INFO - 21/12/19 12:35:36 INFO ResourceUtils: ==============================================================
[2021-12-19 12:35:36,794] {spark_submit.py:514} INFO - 21/12/19 12:35:36 INFO SparkContext: Submitted application: Get Ads per county
[2021-12-19 12:35:36,878] {spark_submit.py:514} INFO - 21/12/19 12:35:36 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(cores -> name: cores, amount: 1, script: , vendor: , memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2021-12-19 12:35:36,910] {spark_submit.py:514} INFO - 21/12/19 12:35:36 INFO ResourceProfile: Limiting resource is cpu
[2021-12-19 12:35:36,951] {spark_submit.py:514} INFO - 21/12/19 12:35:36 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2021-12-19 12:35:37,314] {spark_submit.py:514} INFO - 21/12/19 12:35:37 INFO SecurityManager: Changing view acls to: ***
[2021-12-19 12:35:37,317] {spark_submit.py:514} INFO - 21/12/19 12:35:37 INFO SecurityManager: Changing modify acls to: ***
[2021-12-19 12:35:37,329] {spark_submit.py:514} INFO - 21/12/19 12:35:37 INFO SecurityManager: Changing view acls groups to:
[2021-12-19 12:35:37,360] {spark_submit.py:514} INFO - 21/12/19 12:35:37 INFO SecurityManager: Changing modify acls groups to:
[2021-12-19 12:35:37,401] {spark_submit.py:514} INFO - 21/12/19 12:35:37 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users  with view permissions: Set(***); groups with view permissions: Set(); users  with modify permissions: Set(***); groups with modify permissions: Set()
[2021-12-19 12:35:40,956] {spark_submit.py:514} INFO - 21/12/19 12:35:40 INFO Utils: Successfully started service 'sparkDriver' on port 44281.
[2021-12-19 12:35:41,146] {spark_submit.py:514} INFO - 21/12/19 12:35:41 INFO SparkEnv: Registering MapOutputTracker
[2021-12-19 12:35:41,325] {spark_submit.py:514} INFO - 21/12/19 12:35:41 INFO SparkEnv: Registering BlockManagerMaster
[2021-12-19 12:35:41,487] {spark_submit.py:514} INFO - 21/12/19 12:35:41 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2021-12-19 12:35:41,492] {spark_submit.py:514} INFO - 21/12/19 12:35:41 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2021-12-19 12:35:41,516] {spark_submit.py:514} INFO - 21/12/19 12:35:41 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2021-12-19 12:35:41,710] {spark_submit.py:514} INFO - 21/12/19 12:35:41 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-ce8b070f-e226-40ea-a82c-d1788c5b9474
[2021-12-19 12:35:41,858] {spark_submit.py:514} INFO - 21/12/19 12:35:41 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2021-12-19 12:35:41,946] {spark_submit.py:514} INFO - 21/12/19 12:35:41 INFO SparkEnv: Registering OutputCommitCoordinator
[2021-12-19 12:35:50,123] {spark_submit.py:514} INFO - 21/12/19 12:35:50 WARN Utils: Service 'SparkUI' could not bind on port 4040. Attempting port 4041.
[2021-12-19 12:35:50,131] {spark_submit.py:514} INFO - 21/12/19 12:35:50 WARN Utils: Service 'SparkUI' could not bind on port 4041. Attempting port 4042.
[2021-12-19 12:35:50,154] {spark_submit.py:514} INFO - 21/12/19 12:35:50 WARN Utils: Service 'SparkUI' could not bind on port 4042. Attempting port 4043.
[2021-12-19 12:35:50,214] {spark_submit.py:514} INFO - 21/12/19 12:35:50 INFO Utils: Successfully started service 'SparkUI' on port 4043.
[2021-12-19 12:35:51,107] {spark_submit.py:514} INFO - 21/12/19 12:35:51 INFO SparkUI: Bound SparkUI to 0.0.0.0, and started at http://358a7891c658:4043
[2021-12-19 12:35:55,672] {spark_submit.py:514} INFO - 21/12/19 12:35:55 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark:7077...
[2021-12-19 12:35:56,779] {spark_submit.py:514} INFO - 21/12/19 12:35:56 INFO TransportClientFactory: Successfully created connection to spark/172.20.0.4:7077 after 464 ms (15 ms spent in bootstraps)
[2021-12-19 12:35:57,456] {spark_submit.py:514} INFO - 21/12/19 12:35:57 INFO StandaloneSchedulerBackend: Connected to Spark cluster with app ID app-20211219123557-0018
[2021-12-19 12:35:57,730] {spark_submit.py:514} INFO - 21/12/19 12:35:57 INFO Utils: Successfully started service 'org.apache.spark.network.netty.NettyBlockTransferService' on port 46201.
[2021-12-19 12:35:57,746] {spark_submit.py:514} INFO - 21/12/19 12:35:57 INFO NettyBlockTransferService: Server created on 358a7891c658:46201
[2021-12-19 12:35:57,757] {spark_submit.py:514} INFO - 21/12/19 12:35:57 INFO BlockManager: Using org.apache.spark.storage.RandomBlockReplicationPolicy for block replication policy
[2021-12-19 12:35:57,916] {spark_submit.py:514} INFO - 21/12/19 12:35:57 INFO BlockManagerMaster: Registering BlockManager BlockManagerId(driver, 358a7891c658, 46201, None)
[2021-12-19 12:35:58,101] {spark_submit.py:514} INFO - 21/12/19 12:35:58 INFO BlockManagerMasterEndpoint: Registering block manager 358a7891c658:46201 with 434.4 MiB RAM, BlockManagerId(driver, 358a7891c658, 46201, None)
[2021-12-19 12:35:58,116] {spark_submit.py:514} INFO - 21/12/19 12:35:58 INFO BlockManagerMaster: Registered BlockManager BlockManagerId(driver, 358a7891c658, 46201, None)
[2021-12-19 12:35:58,121] {spark_submit.py:514} INFO - 21/12/19 12:35:58 INFO BlockManager: Initialized BlockManager: BlockManagerId(driver, 358a7891c658, 46201, None)
[2021-12-19 12:36:02,044] {spark_submit.py:514} INFO - 21/12/19 12:36:02 INFO StandaloneSchedulerBackend: SchedulerBackend is ready for scheduling beginning after reached minRegisteredResourcesRatio: 0.0
[2021-12-19 12:36:02,452] {spark_submit.py:514} INFO - /home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py:238: FutureWarning: Python 3.6 support is deprecated in Spark 3.2.
[2021-12-19 12:36:02,455] {spark_submit.py:514} INFO - FutureWarning
[2021-12-19 12:36:06,541] {spark_submit.py:514} INFO - 21/12/19 12:36:06 INFO SharedState: Setting hive.metastore.warehouse.dir ('null') to the value of spark.sql.warehouse.dir.
[2021-12-19 12:36:08,001] {spark_submit.py:514} INFO - 21/12/19 12:36:07 INFO SharedState: Warehouse path is 'file:/opt/***/spark-warehouse'.
[2021-12-19 12:36:38,026] {spark_submit.py:514} INFO - running...
[2021-12-19 12:36:46,465] {spark_submit.py:514} INFO - running...
[2021-12-19 12:37:03,364] {spark_submit.py:514} INFO - <urlopen error [Errno -3] Temporary failure in name resolution>
[2021-12-19 12:37:08,556] {spark_submit.py:514} INFO - Traceback (most recent call last):
[2021-12-19 12:37:09,688] {spark_submit.py:514} INFO - File "/opt/***/jobs/get_ads_per_county.py", line 17, in <module>
[2021-12-19 12:37:14,748] {spark_submit.py:514} INFO - get_rent_parse_county_json(county)
[2021-12-19 12:37:14,930] {spark_submit.py:514} INFO - File "/opt/***/jobs/my_packages/fptools/fptools.py", line 191, in h
[2021-12-19 12:37:15,489] {spark_submit.py:514} INFO - return g(x, f(x))
[2021-12-19 12:37:15,498] {spark_submit.py:514} INFO - File "/opt/***/jobs/my_packages/main.py", line 32, in run_get_ads_by_county
[2021-12-19 12:37:15,500] {spark_submit.py:514} INFO - get_daft)(limit, county=county)
[2021-12-19 12:37:15,501] {spark_submit.py:514} INFO - File "/opt/***/jobs/my_packages/scraptools/scrap_main.py", line 252, in get_adds_by_page
[2021-12-19 12:37:15,535] {spark_submit.py:514} INFO - res.append(extract_and_get_data_ads(get_daft, url, county)(daft))
[2021-12-19 12:37:15,537] {spark_submit.py:514} INFO - File "/opt/***/jobs/my_packages/scraptools/scrap_main.py", line 193, in h
[2021-12-19 12:37:15,553] {spark_submit.py:514} INFO - return g(*args)(f(x))
[2021-12-19 12:37:15,779] {spark_submit.py:514} INFO - File "/opt/***/jobs/my_packages/scraptools/scrap_main.py", line 181, in get_data_ads
[2021-12-19 12:37:15,934] {spark_submit.py:514} INFO - res.append(parse_advert_from_daft(ad))
[2021-12-19 12:37:16,878] {spark_submit.py:514} INFO - File "/opt/***/jobs/my_packages/scraptools/scrap_main.py", line 120, in parse_advert
[2021-12-19 12:37:16,883] {spark_submit.py:514} INFO - ad_data = get_ad_data(ad)
[2021-12-19 12:37:16,884] {spark_submit.py:514} INFO - File "/opt/***/jobs/my_packages/fptools/fptools.py", line 184, in format_compose
[2021-12-19 12:37:16,885] {spark_submit.py:514} INFO - vals = [f(x) for f in funcs]
[2021-12-19 12:37:16,886] {spark_submit.py:514} INFO - File "/opt/***/jobs/my_packages/fptools/fptools.py", line 184, in <listcomp>
[2021-12-19 12:37:16,889] {spark_submit.py:514} INFO - vals = [f(x) for f in funcs]
[2021-12-19 12:37:17,396] {spark_submit.py:514} INFO - File "/opt/***/jobs/my_packages/fptools/fptools.py", line 35, in h
[2021-12-19 12:37:19,587] {spark_submit.py:514} INFO - return g(f(x))
[2021-12-19 12:37:19,740] {spark_submit.py:514} INFO - File "/opt/***/jobs/my_packages/scraptools/scrap_info.py", line 37, in extract_id
[2021-12-19 12:37:19,913] {spark_submit.py:514} INFO - return ad.find('p', {'class': re.compile('DaftIDText.')}).get_text()
[2021-12-19 12:37:20,414] {spark_submit.py:514} INFO - AttributeError: 'NoneType' object has no attribute 'find'
[2021-12-19 12:37:28,912] {spark_submit.py:514} INFO - 21/12/19 12:37:28 INFO SparkContext: Invoking stop() from shutdown hook
[2021-12-19 12:37:29,429] {spark_submit.py:514} INFO - 21/12/19 12:37:29 INFO SparkUI: Stopped Spark web UI at http://358a7891c658:4043
[2021-12-19 12:37:29,562] {spark_submit.py:514} INFO - 21/12/19 12:37:29 INFO StandaloneSchedulerBackend: Shutting down all executors
[2021-12-19 12:37:29,606] {spark_submit.py:514} INFO - 21/12/19 12:37:29 INFO CoarseGrainedSchedulerBackend$DriverEndpoint: Asking each executor to shut down
[2021-12-19 12:37:29,868] {spark_submit.py:514} INFO - 21/12/19 12:37:29 INFO MapOutputTrackerMasterEndpoint: MapOutputTrackerMasterEndpoint stopped!
[2021-12-19 12:37:30,387] {spark_submit.py:514} INFO - 21/12/19 12:37:30 INFO MemoryStore: MemoryStore cleared
[2021-12-19 12:37:30,408] {spark_submit.py:514} INFO - 21/12/19 12:37:30 INFO BlockManager: BlockManager stopped
[2021-12-19 12:37:30,505] {spark_submit.py:514} INFO - 21/12/19 12:37:30 INFO BlockManagerMaster: BlockManagerMaster stopped
[2021-12-19 12:37:30,531] {spark_submit.py:514} INFO - 21/12/19 12:37:30 INFO OutputCommitCoordinator$OutputCommitCoordinatorEndpoint: OutputCommitCoordinator stopped!
[2021-12-19 12:37:31,161] {spark_submit.py:514} INFO - 21/12/19 12:37:31 INFO SparkContext: Successfully stopped SparkContext
[2021-12-19 12:37:31,181] {spark_submit.py:514} INFO - 21/12/19 12:37:31 INFO ShutdownHookManager: Shutdown hook called
[2021-12-19 12:37:31,184] {spark_submit.py:514} INFO - 21/12/19 12:37:31 INFO ShutdownHookManager: Deleting directory /tmp/spark-002f7b35-30b8-44f4-98ad-d0ec104519ea/pyspark-7c839a3e-0024-4161-844d-6a70ba1f02af
[2021-12-19 12:37:31,279] {spark_submit.py:514} INFO - 21/12/19 12:37:31 INFO ShutdownHookManager: Deleting directory /tmp/spark-cdfbc0ec-c008-42b8-b968-067c99d0fa15
[2021-12-19 12:37:31,310] {spark_submit.py:514} INFO - 21/12/19 12:37:31 INFO ShutdownHookManager: Deleting directory /tmp/spark-002f7b35-30b8-44f4-98ad-d0ec104519ea
[2021-12-19 12:37:36,803] {taskinstance.py:1703} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1332, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1458, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1514, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 179, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 446, in submit
    f"Cannot execute: {self._mask_cmd(spark_submit_cmd)}. Error code is: {returncode}."
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://spark:7077 --conf spark.master=spark://spark:7077 --name scrap data --verbose /opt/***/jobs/get_ads_per_county.py longford. Error code is: 1.
[2021-12-19 12:37:36,906] {taskinstance.py:1280} INFO - Marking task as UP_FOR_RETRY. dag_id=scraping_daft_longford, task_id=scrap_data, execution_date=20211218T101000, start_date=20211219T123438, end_date=20211219T123736
[2021-12-19 12:37:37,359] {standard_task_runner.py:91} ERROR - Failed to execute job 3190 for task scrap_data
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/task/task_runner/standard_task_runner.py", line 85, in _start_by_fork
    args.func(args, dag=self.dag)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/cli_parser.py", line 48, in command
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/utils/cli.py", line 92, in wrapper
    return f(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 292, in task_run
    _run_task_by_selected_method(args, dag, ti)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 107, in _run_task_by_selected_method
    _run_raw_task(args, ti)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 184, in _run_raw_task
    error_file=args.error_file,
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/utils/session.py", line 70, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1332, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1458, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1514, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 179, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 446, in submit
    f"Cannot execute: {self._mask_cmd(spark_submit_cmd)}. Error code is: {returncode}."
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://spark:7077 --conf spark.master=spark://spark:7077 --name scrap data --verbose /opt/***/jobs/get_ads_per_county.py longford. Error code is: 1.
[2021-12-19 12:37:37,677] {local_task_job.py:154} INFO - Task exited with return code 1
[2021-12-19 12:37:38,647] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
